common/components/player-component.ts

```diff
@@ -258,6 +258,12 @@ export class PlayerComponent {
 		// Can't change to existing active row
 		if (newRow === currentActiveRow) return false
 
+		// Can't change to opponent's row
```

Comment:
It would be better not to have a comment before an assert. The assertion here already tells us what the comment does and its MORE POWERFUL :rocket:  because if anything changes in the game and our comment is wrong, the assertion will then fail.
Regular comments don't have this power so its best to use them only if you can't assert the result or printing out what the comment says in a log doesn't make sense either.